### ISSUE TITLE
Put all Msysgit's right-click context menu options in a sub-menu in Windows Exporer?

### ISSUE THREAD

**Issue by laggingreflex** [msysgit/Git-Cheetah#20]
Is it possible (if not, please make it a feature-request) to put all Msysgit's right-click menu options in a <strong><em>sub-</em></strong>menu?

All of the options makes my menu enormous

![][1]

[1]:http://i.imgur.com/W7XXRTd.png

**Comment by dscho** [comment]
It is possible! Please have a crack at it yourself. It is [easy to build Git Cheetah (the Explorer extension)](https://github.com/msysgit/msysgit/wiki/GitCheetah#How_to_compileinstall_it_in_msysGit).

**Comment by kusma** [comment]
Here's an ancient patch for that, perhaps it can still be applied:
https://groups.google.com/d/msg/msysgit/EIte82yKOIo/vonruGjvvX0J

The comment about finder lacking submenus does not apply any more, IIRC.

**Comment by dscho** [comment]
@laggingreflex did you plan on working on this issue, or can I close this ticket?

**Comment by laggingreflex** [comment]
Lazy me never actually got around to it. (not being familiar with the language doesn't help either) I guess I'll check it out later if need be, thanks to the resources you guys have pointed to, so this may be closed for now. Also since I've switched to 2.5 I notice there are far fewer context menu options now. I only selected "Git Bash" during the install which gives only 1 option and I'm happy with just that.
